### DISCUSSION TITLE
MySQL 8.x - Remote connection does not work

### DISCUSSION THREAD

**Discussion by hedbonker** [Homebrew/discussions#4745] [Getting started]
### Output of `brew config`

```shell
Not applicable
```

### Output of `brew doctor`

```shell
Not applicable
```

### Description of issue

I defined a user on mysql with full privileges.  I edited /usr/local/etc/my.cnf file and set the bind-address to 0.0.0.0.  But when I try to connect with workbench from another machine (these are all on my local network with IP addrs like 192.168.0.XXX) the connection is always rejected.  I can ping the machine just fine from my remote machine.

Any tips on getting this to work?

**Comment by SMillerDev** [comment]
Try seeing if macOS even allows that connection

**Reply by hedbonker** [reply]
I will check that.  Thanks.
